styles/styles.css

```diff
@@ -0,0 +1,582 @@
+  font-family: 'Neuton', sans-serif;
+  color: #093c60;
+  font-size: 1.4rem;
+  margin: 0;
+}
+
+h3 {
+  font-family: 'Neuton', sans-serif;
+  color: #093c60;
+  font-size: 1.2rem;
+  margin: 0;
+}
+
+h4,
+h5,
+h6 {
+  font-family: 'Neuton', sans-serif;
+  color: #093c60;
+  margin: 0;
+}
+
+
+/* skip link */
+
+.hidden {
+  position: relative;
+  left: -10000px;
+  top: auto;
+}
+
+.hidden:focus {
+  left: 0;
+}
+
+
+/* header */
+
+header {
+  background-image: url("../images/header-image.jpg");
+  background-size: cover;
+  padding: 2rem 6rem 6rem;
+}
+
+.header-headline {
+  color: #81b741;
+  font-style: italic;
+  line-height: .2rem;
+  margin: 0;
+}
+
+.flex-header {
+  display: flex;
+  height: 17rem;
+  flex-wrap: wrap;
+}
+
+.flex-header-1 {
+  flex: 1;
+  align-items: flex-start;
+}
+
+.flex-header-1 img {
+  width: 18.5rem;
+}
+
+.flex-header-2 {
+  margin-left: auto;
+}
+
+.flex-cart {
+  display: flex;
+  height: 3rem;
+  width: 100px;
+  flex-wrap: nowrap;
+}
+
+.flex-cart img {
+  width: 20px;
+}
+
+.flex-icon {
+  flex: 25px;
+  padding: 0px 15px 10px;
+}
+
+.flex-price {
+  flex: 30px;
+  color: white;
+}
+
+.green-button {
+  margin: 0;
+  color: white;
+  font-weight: 600;
+  text-align: center;
+  background-color: #81b741;
+  height: 2rem;
+  width: 100%;
+  padding-top: .3rem;
+}
+
+
+/* top navigation */
+
+main nav:first-of-type {
+  background: #ececec;
+  box-shadow: 0rem .2rem .7rem #c9c9c9;
+  padding-left: 6rem;
+}
+
+nav {
+  font-weight: 600;
+}
+
+.flex-nav {
+  display: flex;
+  width: 250px;
+  height: 3rem;
+  flex-wrap: nowrap;
+  padding: .8rem 0rem 1.5rem;
+}
+
+.flex-nav-home {
+  flex: 1.5rem;
+}
+
+.flex-nav-dogs {
+  flex: 1.5rem;
+}
+
+.flex-nav-blog {
+  flex: 3rem;
+}
+
+/* add slash after nav menu items */
+
+.flex-nav-home a::after {
+  content: "/";
+}
+
+.flex-nav-dogs a::after {
+  content: "/";
+}
+
+
+/* dogs grid layout */
+
+.main-container {
+  display: grid;
+  grid-template-columns: 9% 9% 9% 9% 9% 9% 9% 9%;
+  grid-template-rows: auto auto auto auto auto;
+  grid-column-gap: 2rem;
+  width: 100%;
+  padding: 0 6rem 4rem 6rem;
+}
+
+.grid-main-heading {
+  grid-column: 1 / 9;
+  grid-row: 1 / 2;
+  padding-top: 2rem;
+}
+
+.grid-main1 {
+  grid-column: 1 / 4;
+  grid-row: 1 / 2;
+  padding-top: 2rem;
+}
+
+.grid-main2 {
+  grid-column: 4 / 9;
+  grid-row: 1 / 2;
+  padding-top: 2rem;
+}
+
+.grid-main3 {
+  grid-column: 1 / 3;
+  grid-row: 2 / 3;
+  padding-top: 2rem;
+}
+
+.grid-main4 {
+  grid-column: 3 / 5;
+  grid-row: 2 / 3;
+  padding-top: 2rem;
+}
+
+.grid-main5 {
+  grid-column: 5 / 7;
+  grid-row: 2 / 3;
+  padding-top: 2rem;
+}
+
+.grid-main6 {
+  grid-column: 7 / 9;
+  grid-row: 2 / 3;
+  padding-top: 2rem;
+}
+
+.grid-main7 {
+  grid-column: 1 / 3;
+  grid-row: 3 / 4;
+  padding-top: 3rem;
+}
+
+.grid-main8 {
+  grid-column: 3 / 5;
+  grid-row: 3 / 4;
+  padding-top: 3rem;
+}
+
+.grid-main9 {
+  grid-column: 5 / 7;
+  grid-row: 3 / 4;
+  padding-top: 3rem;
+}
+
+.grid-main10 {
```

Comment:
These numbered classes aren't super scaleable. What if you had 1000000 items? You could use one class for each of these tiles. Obviously you'd have to rethink how this works, but you want this to be as flexible as possible.